papaya-core/src/main/java/pl/edu/agh/papaya/service/sprint/SprintCreationWizard.java

```diff
@@ -52,8 +46,7 @@ public Sprint create() {
         sprint.setDurationPeriod(AssertionUtil.require("durationPeriod", durationPeriod));
         sprint.setEnrollmentPeriod(AssertionUtil.require("enrollmentPeriod", enrollmentPeriod));
         sprint.setDateClosed(null);
-        sprint.setTimeBurned(null);
-        sprint.setTimePlanned(AssertionUtil.require("timePlanned", timePlanned));
+        sprint.setStats(new SprintStats());
```

Comment:
Jak `Sprint` ma jako pole `private SprintStats stats = new SprintStats();`, to to nam coś daje?

Reply:
nie, ale przywołałem dobre pytanie - gdzie lepiej?

Reply:
zostawie `private SprintStats stats = new SprintStats();` zeby nie bylo niespodzianek